docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -158,19 +150,18 @@ To check the Subscription cleaned Event type, run:
 kubectl get subscriptions lastorder-payment-sub -o=jsonpath="{.status.cleanEventTypes}"
 ```
 
-Now, note that the returned cleaned event type `["sap.kyma.custom.myapp.order.paymentsuccess.v1"]` does not contain the hyphen (i.e. `-`) in `payment-success` part, because Eventing services cleans out the non-alphanumeric characters from the Event name and uses the cleaned Event name in underlying Eventing-Backend.
+Note that the returned cleaned event type `["sap.kyma.custom.myapp.order.paymentsuccess.v1"]` does not contain the hyphen `-` in the `payment-success` part. That's because Kyma Eventing cleans out the non-alphanumeric characters from the Event name and uses the cleaned Event name in the underlying Eventing backend.
```

Comment:
I think we don't need "cleaned" here, the following sentence explains it. 

```suggestion
Note that the returned event type `["sap.kyma.custom.myapp.order.paymentsuccess.v1"]` does not contain the hyphen `-` in the `payment-success` part. That's because Kyma Eventing cleans out the non-alphanumeric characters from the Event name and uses the cleaned Event name in the underlying Eventing backend.
```